src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:3000/api"
+      method="post"
+      class="searchform"
```

Comment:
use either `camelCase` or `kebab-case` for your classnames. It helps to keep your code consistent and easy to read

Reply:
Fixed.